technic/register.lua

```diff
@@ -34,6 +34,8 @@ function technic.register_power_tool(itemname, itemdef)
 	itemdef.wear_represents = itemdef.wear_represents or "technic_RE_charge"
 	itemdef.technic_max_charge = max_charge
 	itemdef.technic_wear_factor = 65535 / max_charge
+	itemdef.technic_get_charge = itemdef.technic_get_charge or technic.get_RE_charge
+	itemdef.technic_set_charge = itemdef.technic_set_charge or technic.set_RE_charge
 	itemdef.on_refill = itemdef.on_refill or function(stack)
 		-- This is always using originally defined max_charge even if stack somehow changed to another
 		technic.set_RE_charge(stack, max_charge)
```

Comment:
This should also be using `itemdef.technic_set_charge` instead of `technic.set_RE_charge`

Reply:
f5ef419